components/TokenView/samples/TokenViewItemsSourceSample.xaml

```diff
@@ -0,0 +1,35 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="TokenViewExperiment.Samples.TokenViewItemsSourceSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+      xmlns:local="using:TokenViewExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <StackPanel Orientation="Vertical"
+                Spacing="12">
+        <labs:TokenView CanRemoveTokens="{x:Bind CanRemoveTokens, Mode=OneWay}"
+                        ItemsSource="{x:Bind MyDataSet, Mode=OneWay}"
+                        SelectedIndex="1"
+                        SelectionChanged="TokenView_SelectionChanged"
+                        SelectionMode="Single">
+            <labs:TokenView.ItemTemplate>
+                <DataTemplate x:DataType="local:MyDataModel">
+                    <Grid>
+                        <TextBlock Margin="16,0,16,0"
+                                   Style="{StaticResource CaptionTextBlockStyle}"
+                                   Text="{x:Bind Name}" />
+                    </Grid>
+                </DataTemplate>
```

Comment:
Just wondering if we want this to work like the new-style where you specify the item? That way you can bind to content and/or icon? Otherwise you have to re-style the icon yourself here in this scenario?

That's a bigger change though and as we know has complications potentially. So maybe something to think about for next big iteration from any other feedback we get?